src/app/app-routing.module.ts

```diff
@@ -1,6 +1,7 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 
+import { IsLoggedInGuard } from './auth/guards/is-logged-in.guard';
```

Comment:
@k1k8r will be better to use aliases for the guards in the module. What do you think about it? Something like this: `@auth/guards`. It makes your code more clear and readable.